### PR TITLE
Exchange SERVICE and NAME column in kn revision list output

### LINKED ISSUE #195: Exchange SERVICE and NAME column in kn revision list output
The NAME should be always first, and only then the service name to which a revision belongs. This not only for consistencies sake, also to make the first column to be the id (and since a revision name typically is prefixed with the service name, you have both info already in the first column)

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #195

### CODE REVIEW

File: pkg/kn/commands/revision/human_readable_flags.go [hunk 1]
```diff
@@ -52,10 +52,10 @@ func printRevisionList(revisionList *servingv1alpha1.RevisionList, options hprin
 	return rows, nil
 }
 
-// printRevision populates the knative revision table rows
+// printRevision populates the Knative revision table rows
 func printRevision(revision *servingv1alpha1.Revision, options hprinters.PrintOptions) ([]metav1beta1.TableRow, error) {
-	service := revision.Labels[serving.ConfigurationLabelKey]
 	name := revision.Name
+	service := revision.Labels[serving.ConfigurationLabelKey]
```
Comment (rhuss) [inline pkg/kn/commands/revision/human_readable_flags.go]: I'm still not sold that it should be the `ConfigurationLabelKey`. It's just by accident that the Configuration is named like the service. You should use `ServiceLabelKey` instead.

Comment (navidshaikh) [reply pkg/kn/commands/revision/human_readable_flags.go]:
```
  // ServiceLabelKey is the label key attached to a Route and Configuration indicating by
  // which Service they are created.
  ServiceLabelKey = GroupName + "/service"
```
@rhuss : yup, updating..

**Pull request merged**